components/esp_eth/src/w5500.h

```diff
@@ -16,6 +16,8 @@
 #define W5500_BSB_OFFSET  (3)  // Block Select Bits offset
 #define W5500_RWB_OFFSET  (2)  // Read Write Bits offset
 
+#define W5500_CHIP_VERSION 0x4 // Chip version that VERSIONR returns
```

Comment:
Please follow the coding style here.
```suggestion
#define W5500_CHIP_VERSION (0x4) // Chip version that VERSIONR returns
```